### PR TITLE
Jan11 Description Updates

### PR DESCRIPTION
**Pull request by mmthomaslinode**
More updates from the Victorious sheet.

### CODE REVIEW

File: docs/guides/applications/configuration-management/puppet/use-puppet-modules-to-create-a-lamp-stack/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ slug: use-puppet-modules-to-create-a-lamp-stack
 author:
     name: Elle Krout
     email: ekrout@linode.com
-description: 'Learn how to efficiently use Puppet modules to manage files and services, create templates, and store data in Hiera in this simple tutorial.'
+description: 'With this guude, you will learn how to efficiently use Puppet modules to manage files and services, create templates, and store data in Hiera on Ubuntu 14.04 LTS.'
```
Comment (andystevensname) [inline docs/guides/applications/configuration-management/puppet/use-puppet-modules-to-create-a-lamp-stack/index.md:6]: guide misspelled

File: docs/guides/databases/mysql/connect-metabase-with-mysql-for-data-exploration/index.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ slug: connect-metabase-with-mysql-for-data-exploration
 author:
   name: Sam Foo
   email: docs@linode.com
-description: 'Metabase is a data exploration tool that makes analytics accessible to everyone even without knowledge of SQL. Connect Metabase to a local MySQL server and deploy through a reverse proxy on NGINX using this guide.'
-og_description: 'Metabase provides a clean interface to query data on your browser. Metabase offers functionality to analyze data without SQL, create dashboards, and track metrics. This guide shows how to connect MySQL to Metabase then deploy on NGINX through a reverse proxy'
+description: 'This guide shows you how to connect Metabase, a datae xploration tool that makes analytics easy, to a local MySQL server and deploy through a reverse proxy on NGINX.'
```
Comment (andystevensname) [inline docs/guides/databases/mysql/connect-metabase-with-mysql-for-data-exploration/index.md:6]: data exploration

File: docs/guides/databases/mysql/how-to-create-source-replica-replication-in-mysql/index.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 slug: configure-source-replica-replication-in-mysql
 author:
   name: Jeff Novotny
-description: 'The popular MySQL database application is more reliable if replication is configured. This guide explains how to configure source-replica data replication in MySQL.'
+description: 'This guide explains how to configure source-replica data replication in the popula rmySQL database application.'
```
Comment (andystevensname) [inline docs/guides/databases/mysql/how-to-create-source-replica-replication-in-mysql/index.md:5]: popular MySQL

File: docs/guides/development/javascript/authenticating-over-websockets-jwt/index.md [hunk 4]
```diff
@@ -2,8 +2,8 @@
 slug: authenticating-over-websockets-with-jwt
 author:
   name: Nathaniel Stickman
-description: "WebSockets enable real-time communications between clients and server and they can be immensely useful in a variety of web applications. JSON Web Tokens can provide lightweight and secure authentication. They pair well with WebSockets to ensure connections are authorized. In this guide, you learn more about both WebSockets and JSON Web Tokens in this guide. You can also follow an example showing you how to use them together in your application."
-og_description: "WebSockets enable real-time communications between clients and server and they can be immensely useful in a variety of web applications. JSON Web Tokens can provide lightweight and secure authentication. They pair well with WebSockets to ensure connections are authorized. In this guide, you learn more about both WebSockets and JSON Web Tokens in this guide. You can also follow an example showing you how to use them together in your application."
+description: "In this guide, you learn more about both WebSockets and JSON Web Tokens will also follow an example showing you how to use them together in your app."
```
Comment (andystevensname) [inline docs/guides/development/javascript/authenticating-over-websockets-jwt/index.md:5]: This sentence construction is not correct. You could write something like "...JSON Web Tokens. Follow an example..."

File: docs/guides/development/javascript/how-to-authenticate-using-jwt/index.md [hunk 5]
```diff
@@ -2,8 +2,8 @@
 slug: how-to-authenticate-using-jwt
 author:
   name: Nathaniel Stickman
-description: "JSON Web Tokens (JWTs) provide a compact and secure standard for sending information, especially on the web. This guide explains the key concepts behind JWTs and walks you through getting started implementing your own using Express."
-og_description: "JSON Web Tokens (JWTs) provide a compact and secure standard for sending information, especially on the web. This guide explains the key concepts behind JWTs and walks you through getting started implementing your own using Express."
+description: "In this guide, you will learn key concetps behing the concept of JSON Web Tokens (JWTs) and how they provide a compact and secure standard for sending information, especially on the web."
```
Comment (andystevensname) [inline docs/guides/development/javascript/how-to-authenticate-using-jwt/index.md:5]: concepts behind JSON Web Tokens

File: docs/guides/game-servers/host-a-terraria-server-on-your-linode/index.md [hunk 6]
```diff
@@ -3,8 +3,8 @@ slug: host-a-terraria-server-on-your-linode
 author:
   name: Linode Community
   email: docs@linode.com
-description: 'Terraria is a two-dimensional sandbox game similar to Minecraft that allows players to explore, build, and battle in an open world. This guide will outline everything required to run a Terraria server for yourself or others to play on'
-og_description: 'Run a Terraria server for yourself and your friends to play on. This guide will teach you setup and configuration for Linux distributions.'
+description: 'In this guide, you will learn how to install and configure a Terraria, a two-dimensional sandbox game similar to Minecraft, on a Linode.'
```
Comment (andystevensname) [inline docs/guides/game-servers/host-a-terraria-server-on-your-linode/index.md:6]: configure Terraria (no a)

File: docs/guides/platform/manager/an-overview-of-the-linode-cloud-manager/index.md [hunk 7]
```diff
@@ -3,7 +3,7 @@ slug: an-overview-of-the-linode-cloud-manager
 author:
   name: Linode
   email: docs@linode.com
-description: 'This guide provides an overview of the Linode Cloud Manager. You will find information about each section of the Cloud Manager, specific features that you may need help finding, and settings that might make your overall Cloud Manager experience better.'
+description: 'This guide provides an overview of the Linode Cloud Manager and covers each specific features that you may need help finding.'
```
Comment (andystevensname) [inline docs/guides/platform/manager/an-overview-of-the-linode-cloud-manager/index.md:6]: covers specific features (no each)

File: docs/guides/platform/marketplace/chevereto-marketplace-app/index.md [hunk 8]
```diff
@@ -3,7 +3,7 @@ slug: chevereto-marketplace-app
 author:
   name: Linode
   email: docs@linode.com
-description: "Chevereto is a modern image sharing solution with drag-and-drop upload experience, multi-server support, moderation tools, categories, user accounts, private albums, and more. This tutorial walks you through deploying Chevereto using the Linode Marketplace."
+description: "This guide shows you how to install chevereto, a modern image sharing solution with drag-and-drop upload experience and more using the Linode One-Click Marketplace."
```
Comment (andystevensname) [inline docs/guides/platform/marketplace/chevereto-marketplace-app/index.md:6]: Capitalize Chevereto

File: docs/guides/security/basics/docker-security-essentials/index.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 slug: docker-security-essentials
 author:
   name: Hackersploit
-description: "Learn about the basics of securing Docker containers, including managing user acess, preventing privilege escalation attacks, and limiting kernel capabilities."
+description: "This guide will show you the basics of securing Docker containers, including managing user access, preventing privliges escalations, and more."
```
Comment (andystevensname) [inline docs/guides/security/basics/docker-security-essentials/index.md:5]: spelling of privilege (singular)

File: docs/guides/security/basics/how-to-configure-automated-security-updates-centos/index.md [hunk 10]
```diff
@@ -2,7 +2,7 @@
 slug: how-to-configure-automated-security-updates-centos
 author:
   name: Hackersploit
-description: "Looking for better management of security updates on CentOS? Learn how to use the dnf-automatic tool or the Cockpit web application to manage these updates."
+description: "In this tutorial, you will learn how to configure automated updates on CentOS using the dnf-automatic tool or the COckpit web application."
```
Comment (andystevensname) [inline docs/guides/security/basics/how-to-configure-automated-security-updates-centos/index.md:5]: No cap O in Cockpit

File: docs/guides/security/basics/how-to-configure-automated-security-updates-ubuntu/index.md [hunk 11]
```diff
@@ -2,7 +2,7 @@
 slug: how-to-configure-automated-security-updates-ubuntu
 author:
   name: Hackersploit
-description: "Looking for better management of security updates on Ubuntu? Learn how to use the dnf-automatic tool or the Cockpit web application to manage these updates."
+description: "In this tutorial, you will learn how to configure automated updates on Ubuntu, by using the dns-automatic tool or the Cockpit web application."
```
Comment (andystevensname) [inline docs/guides/security/basics/how-to-configure-automated-security-updates-ubuntu/index.md:5]: Is it dnf-automatic tool, or dns-automatic?

File: docs/guides/web-servers/apache/how-to-configure-http-2-on-apache/index.md [hunk 12]
```diff
@@ -2,8 +2,8 @@
 slug: how-to-configure-http-2-on-apache
 author:
   name: Jeff Novotny
-description: 'HTTP/2 is an update to the original Hypertext Transfer Protocol (HTTP) specification offering improvements in efficiency and latency. This guide explains how to configure and use HTTP/2 on an Apache server and how to test that it is working.'
-og_description: 'HTTP/2 is an update to the original Hypertext Transfer Protocol (HTTP) specification offering improvements in efficiency and latency. This guide explains how to configure and use HTTP/2 on an Apache server and how to test that it is working.'
+description: 'In this tutorial, you will learn how to configure HTTP/2 on Apache using Debian.'
```
Comment (andystevensname) [inline docs/guides/web-servers/apache/how-to-configure-http-2-on-apache/index.md:5]: configure HTTP/2 using Apache on Debian

File: docs/guides/websites/cms/wordpress/configuring-wordpress/index.md [hunk 13]
```diff
@@ -2,8 +2,8 @@
 slug: configuring-wordpress
 author:
   name: Nathaniel Stickman
-description: "You can configure your WordPress site''s default installation to enhance its functionality. For example, install the PHP GD extension to modify images within WordPress. Another common change is to prettify your WordPress site''s permalinks. This guide shows you how to configure your WordPress site to enable helpful utilities."
-og_description:  "You can configure your WordPress site''s default installation to enhance its functionality. For example, install the PHP GD extension to modify images within WordPress. Another common change is to prettify your WordPress site''s permalinks. This guide shows you how to configure your WordPress site to enable helpful utilities."
+description: "This guide shows you how to conifugure your WordPress site with additional utilities, such as installing PHP GD extensions and prettifying your permalinks."
```
Comment (andystevensname) [inline docs/guides/websites/cms/wordpress/configuring-wordpress/index.md:5]: spelling of configure

File: docs/guides/websites/lms/how-to-install-canvas-on-debian-10/index.md [hunk 14]
```diff
@@ -3,7 +3,7 @@ slug: how-to-install-canvas-on-debian-10
 author:
   name: Linode Community
   email: docs@linode.com
-description: 'Canvas is a learning management system that you can use to create a fully-featured website for education or training courses. This guide provides instructions on how you can install Canvas on Debian 10.'
+description: 'This guide will show you how to install the popula rlearning management system Canvas on Debian 10.'
```
Comment (andystevensname) [inline docs/guides/websites/lms/how-to-install-canvas-on-debian-10/index.md:6]: popular learning

**Review by andystevensname — CHANGES_REQUESTED** [review]

**Review by andystevensname — APPROVED** [review]

**Review by wildmanonline — APPROVED** [review]

File: docs/guides/applications/containers/access-an-apache-web-server-inside-a-lxd-container/index.md [hunk 15]
```diff
@@ -3,7 +3,7 @@ slug: access-an-apache-web-server-inside-a-lxd-container
 author:
   name: Linode
   email: docs@linode.com
-description: "Compared to other Containers, LXDmachine containers each work like typical servers. Here's show to run Apache inside LXD and direct traffic to the container."
+description: "This guide will show you how to install and configure Apache Web Server inside of LXD server and then direct web traffic to the container."
```
Comment (andystevensname) [inline docs/guides/applications/containers/access-an-apache-web-server-inside-a-lxd-container/index.md]: LXD container

**Review by andystevensname — APPROVED** [review]

**Pull request merged**